worldengine/hdf5_serialization.py

```diff
@@ -16,87 +18,88 @@ def save_world_to_hdf5(world, filename):
 
     elevation_grp = f.create_group("elevation")
     elevation_ths_grp = elevation_grp.create_group("thresholds")
-    elevation_ths_grp["sea"] = world.elevation['thresholds'][0][1]
-    elevation_ths_grp["plain"] = world.elevation['thresholds'][1][1]
-    elevation_ths_grp["hill"] = world.elevation['thresholds'][2][1]
+    elevation_ths_grp["sea"] = world.layers['elevation'].thresholds[0][1]
+    elevation_ths_grp["plain"] = world.layers['elevation'].thresholds[1][1]
+    elevation_ths_grp["hill"] = world.layers['elevation'].thresholds[2][1]
```

Comment:
Should these maybe saved with a for-loop? In case more are ever added.^^ (applies to the protobuf-save, too)